cross/ccomp-armv7a.txt

```diff
@@ -0,0 +1,13 @@
+[binaries]
+c = 'ccomp'
+ar = 'ccomp'
+strip = 'strip'
+
+[built-in options]
+c_args      = ['-target armv7a-eabi', '-fall']
```

Comment:
Oh, I assume this would be better written as `['-target', 'armv7a-eabi', '-fall']`? @dcbaker?

Reply:
Yes. All command lines should be defined as fully separated arrays, not strings.

Reply:
Okay, adapted